src/main/java/org/spongepowered/api/text/translation/Translation.java

```diff
@@ -24,19 +24,20 @@
  */
 package org.spongepowered.api.text.translation;
 
-import org.spongepowered.api.CatalogType;
-import org.spongepowered.api.text.translation.locale.Locales;
+import java.util.Locale;
 
 /**
  * Represents an identifier for text that can be translated into multiple
- * languages. This is usually sent directly to the client and translated there,
- * not on the server.
+ * languages. Minecraft-included translations are generally translated clientside.
+ * Translations not included in Minecraft are generally expected to
+ * be translated server-side, for example using Gettext or a
+ * {@link java.util.ResourceBundle}
```

Comment:
How does the server detect whether the translation is (or supposted to be) included in the minecraft client?
Some full mods (client and server) may want to resolve the text client side because they also provide resourcepacks. There should probably an interface or a method whether the translation should be resolved on the server side. An internal Set of keys could work as well, but mods should be able to add to it.